modules/ROOT/pages/custom-library-development.adoc

```diff
@@ -0,0 +1,746 @@
+= Develop a custom library
+:description: Develop a custom library project that will be used in a Bonita project and mutualize advanced business rules capabilities.
+
+{description}
+
+== Why should I develop a custom library ?
+
+* To extract advanced logic from processes script expressions
+* To be able to use standard technology to automate testing for this logic
+* To be able to reuse and share this logic between several processes and Bonita projects.
+* To better manage my library dependencies
+
+== Initialize the project
```

Comment:
I suggest to add a 'prerequisite' section before this one, to inform the user that Maven and Java must be installed first, and that some development skills are required.